pkg/cmd/attestation/verification/policy.go

```diff
@@ -49,3 +49,57 @@ func (c EnforcementCriteria) Valid() error {
 	}
 	return nil
 }
+
+func (c EnforcementCriteria) BuildPolicyInformation() string {
+	template :=
+		`
+The following policy criteria will be enforced against all attestations:
+- Predicate type must match %s
+- Signing certificate's OIDC issuer must match %s
+- Artifact must be built in an organization whose URI is %s`
+
+	info := fmt.Sprintf(template, c.PredicateType, c.Certificate.Issuer, c.Certificate.SourceRepositoryOwnerURI)
+
+	if c.Certificate.SourceRepositoryURI != "" {
+		info += fmt.Sprintf("\n- Artifact built in a repository whose URI is %s", c.Certificate.SourceRepositoryURI)
+	}
+
+	if c.Certificate.RunnerEnvironment == GitHubRunner {
+		info += "\n- Attestation must have been generated by an Action workflow executed in a GitHub hosted runner"
```

Comment:
if we say `self-hosted` does that mean the inverse is `GitHub-hosted`?

```suggestion
		info += "\n- Attestation must have been generated by an Action workflow executed in a GitHub-hosted runner"
```

Reply:
If `c.Certificate.RunnerEnvironment` is not set to `GitHubRunner`, it is set to the empty string, which means any runner, self hosted or GitHub hosted.

Reply:
Sorry, I was making a minuscule observation on what we're choosing to hyphenate 😄